2020F-usfca-btec-640.github.io/_config.yml

```diff
@@ -0,0 +1,54 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
```

Comment:
This file should not be a part of the PR. Please remove it so we don't have to deal with merge conflicts.